<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Swordfish</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="VFilePlacesView" name="placesView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="elideMode">
           <enum>Qt::ElideMiddle</enum>
          </property>
          <property name="documentMode">
           <bool>true</bool>
          </property>
          <property name="tabsClosable">
           <bool>true</bool>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QStackedWidget" name="stackedWidget">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="page">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QListView" name="iconView">
                  <property name="contextMenuPolicy">
                   <enum>Qt::CustomContextMenu</enum>
                  </property>
                  <property name="acceptDrops">
                   <bool>true</bool>
                  </property>
                  <property name="editTriggers">
                   <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
                  </property>
                  <property name="dragEnabled">
                   <bool>true</bool>
                  </property>
                  <property name="dragDropMode">
                   <enum>QAbstractItemView::DragDrop</enum>
                  </property>
                  <property name="defaultDropAction">
                   <enum>Qt::CopyAction</enum>
                  </property>
                  <property name="selectionMode">
                   <enum>QAbstractItemView::ExtendedSelection</enum>
                  </property>
                  <property name="textElideMode">
                   <enum>Qt::ElideRight</enum>
                  </property>
                  <property name="movement">
                   <enum>QListView::Snap</enum>
                  </property>
                  <property name="flow">
                   <enum>QListView::LeftToRight</enum>
                  </property>
                  <property name="isWrapping" stdset="0">
                   <bool>true</bool>
                  </property>
                  <property name="resizeMode">
                   <enum>QListView::Adjust</enum>
                  </property>
                  <property name="spacing">
                   <number>6</number>
                  </property>
                  <property name="viewMode">
                   <enum>QListView::IconMode</enum>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                  <property name="selectionRectVisible">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="page_2">
               <layout class="QHBoxLayout" name="horizontalLayout4">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QTreeView" name="treeView">
                  <property name="contextMenuPolicy">
                   <enum>Qt::CustomContextMenu</enum>
                  </property>
                  <property name="acceptDrops">
                   <bool>true</bool>
                  </property>
                  <property name="editTriggers">
                   <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
                  </property>
                  <property name="dragEnabled">
                   <bool>true</bool>
                  </property>
                  <property name="dragDropMode">
                   <enum>QAbstractItemView::DragDrop</enum>
                  </property>
                  <property name="defaultDropAction">
                   <enum>Qt::CopyAction</enum>
                  </property>
                  <property name="alternatingRowColors">
                   <bool>true</bool>
                  </property>
                  <property name="selectionMode">
                   <enum>QAbstractItemView::ExtendedSelection</enum>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>48</width>
                    <height>48</height>
                   </size>
                  </property>
                  <property name="textElideMode">
                   <enum>Qt::ElideRight</enum>
                  </property>
                  <property name="uniformRowHeights">
                   <bool>true</bool>
                  </property>
                  <property name="sortingEnabled">
                   <bool>true</bool>
                  </property>
                  <property name="animated">
                   <bool>true</bool>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                  <property name="expandsOnDoubleClick">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="page_3">
               <layout class="QHBoxLayout" name="horizontalLayout5">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QColumnView" name="columnView">
                  <property name="contextMenuPolicy">
                   <enum>Qt::CustomContextMenu</enum>
                  </property>
                  <property name="acceptDrops">
                   <bool>true</bool>
                  </property>
                  <property name="editTriggers">
                   <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
                  </property>
                  <property name="dragEnabled">
                   <bool>true</bool>
                  </property>
                  <property name="dragDropMode">
                   <enum>QAbstractItemView::DragDrop</enum>
                  </property>
                  <property name="defaultDropAction">
                   <enum>Qt::CopyAction</enum>
                  </property>
                  <property name="alternatingRowColors">
                   <bool>true</bool>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>48</width>
                    <height>48</height>
                   </size>
                  </property>
                  <property name="textElideMode">
                   <enum>Qt::ElideRight</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="contextPane" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewWindow"/>
    <addaction name="actionNewTab"/>
    <addaction name="separator"/>
    <addaction name="actionNewFolder"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionDuplicate"/>
    <addaction name="actionCreateAlias"/>
    <addaction name="separator"/>
    <addaction name="actionSearch"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionSelectAll"/>
    <addaction name="separator"/>
    <addaction name="actionRename"/>
    <addaction name="separator"/>
    <addaction name="actionMoveToTrash"/>
    <addaction name="actionDeletePermanently"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuGo">
    <property name="title">
     <string>&amp;Go</string>
    </property>
    <addaction name="actionBack"/>
    <addaction name="actionForward"/>
    <addaction name="actionParentFolder"/>
    <addaction name="separator"/>
    <addaction name="actionComputer"/>
    <addaction name="actionHome"/>
    <addaction name="actionTrash"/>
    <addaction name="actionNetwork"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionAsIcons"/>
    <addaction name="actionAsDetails"/>
    <addaction name="actionAsColumns"/>
    <addaction name="separator"/>
    <addaction name="actionRearrange"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionContents"/>
    <addaction name="actionReportProblem"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuGo"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBack"/>
   <addaction name="actionForward"/>
   <addaction name="separator"/>
   <addaction name="actionAsIcons"/>
   <addaction name="actionAsDetails"/>
   <addaction name="actionAsColumns"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionNewWindow">
   <property name="icon">
    <iconset theme="window-new">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>New &amp;Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionNewTab">
   <property name="icon">
    <iconset theme="tab-new">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>New &amp;Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionNewFolder">
   <property name="icon">
    <iconset theme="folder-new">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>New &amp;Folder</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset theme="window-close">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionGetInformation">
   <property name="icon">
    <iconset theme="document-properties">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Get Information</string>
   </property>
  </action>
  <action name="actionDuplicate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Duplicate</string>
   </property>
  </action>
  <action name="actionCreateAlias">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Create Alias</string>
   </property>
  </action>
  <action name="actionMoveToTrash">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Mo&amp;ve to Trash</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-undo">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-redo">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-cut">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-copy">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-paste">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="icon">
    <iconset theme="edit-select-all">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionAsIcons">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="view-grid-symbolic">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>As Icons</string>
   </property>
  </action>
  <action name="actionAsDetails">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="view-list-symbolic">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>As Details</string>
   </property>
   <property name="toolTip">
    <string>As Details</string>
   </property>
  </action>
  <action name="actionAsColumns">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="view-list-symbolic">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>As Columns</string>
   </property>
  </action>
  <action name="actionRearrange">
   <property name="text">
    <string>Rearrange</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="go-previous">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Back</string>
   </property>
   <property name="shortcut">
    <string>Alt+Left</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="go-next">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Forward</string>
   </property>
   <property name="shortcut">
    <string>Alt+Right</string>
   </property>
  </action>
  <action name="actionParentFolder">
   <property name="text">
    <string>&amp;Parent Folder</string>
   </property>
   <property name="toolTip">
    <string>Go to the parent folder</string>
   </property>
   <property name="shortcut">
    <string>Alt+Up</string>
   </property>
  </action>
  <action name="actionComputer">
   <property name="icon">
    <iconset theme="computer">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Computer</string>
   </property>
   <property name="toolTip">
    <string>Go to the computer view</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Rename</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionDeletePermanently">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Delete Permanently</string>
   </property>
   <property name="shortcut">
    <string>Shift+Del</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionOpenWithOtherApplication">
   <property name="text">
    <string>Other &amp;Application</string>
   </property>
   <property name="toolTip">
    <string>Open with another application</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset theme="folder-home">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Home</string>
   </property>
   <property name="toolTip">
    <string>Go to home</string>
   </property>
   <property name="shortcut">
    <string>Alt+Home</string>
   </property>
  </action>
  <action name="actionTrash">
   <property name="icon">
    <iconset theme="user-trash">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Trash</string>
   </property>
   <property name="toolTip">
    <string>Go to the trash</string>
   </property>
  </action>
  <action name="actionNetwork">
   <property name="icon">
    <iconset theme="network">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Network</string>
   </property>
   <property name="toolTip">
    <string>Go to the network view</string>
   </property>
  </action>
  <action name="actionContents">
   <property name="text">
    <string>&amp;Contents</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionReportProblem">
   <property name="text">
    <string>Report a &amp;Problem</string>
   </property>
   <property name="toolTip">
    <string>Report a problem</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionArrangeByName">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>By &amp;Name</string>
   </property>
   <property name="toolTip">
    <string>Arrange items by name</string>
   </property>
  </action>
  <action name="actionArrangeBySize">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>By &amp;Size</string>
   </property>
   <property name="toolTip">
    <string>Arrange items by size</string>
   </property>
  </action>
  <action name="actionArrangeByType">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>By &amp;Type</string>
   </property>
   <property name="toolTip">
    <string>Arrange items by type</string>
   </property>
  </action>
  <action name="actionArrangeByModificationDate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>By Modification &amp;Date</string>
   </property>
   <property name="toolTip">
    <string>Arrange items by modification date</string>
   </property>
  </action>
  <action name="actionArrangeReversedOrder">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Re&amp;versed Order</string>
   </property>
   <property name="toolTip">
    <string>Reverse the order</string>
   </property>
  </action>
  <action name="actionConnectToServer">
   <property name="text">
    <string>Connect to &amp;Server...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VFilePlacesView</class>
   <extends>QTreeView</extends>
   <header>VFilePlacesView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
